beets/importer.py

```diff
@@ -1352,7 +1358,26 @@ def emitter(task):
         ])
         return pipeline.multiple(ipl.pull())
 
-    resolve_duplicates(session, task)
+    if type(task) != MergedImportTask:
+        resolve_duplicates(session, task)
+        if task.should_merge_duplicates:
+            duplicate_items = task.duplicate_items(session.lib)
+            for item in duplicate_items:
+                item.id = None
+                item.album_id = None
```

Comment:
This looks like the same rigamarole we have to do when re-importing (which is logical). Instead of duplicating the code to do this, maybe we need a helper function for this.